### ISSUE TITLE
Caldav 0.12-dev0 can't be installed via pip

### ISSUE THREAD

**Issue by Maltimore** [tobixen/calendar-cli#96]
Basically the title. The version requirement of the `caldav` package can't be satisfied currently, so I can't install this (`calendar-cli`) package.

**Comment by tobixen** [comment]
Known issue, the master branch of the calendar-cli requires the development branch of the caldav library for everything to work.  I had hoped to release both caldav 0.12 and the latest calendar-cli before anyone would notice, unfortunately I'm a bit delayed on that.  I'll get to it in the upcoming year, I suppose.

If you use the "legacy" interface in calendar-cli (the calendar-cli command) then caldav 0.11 will suffice.  You may hand-edit the setup.py file and replace 0.12-dev0 with 0.11.

**Comment by Maltimore** [comment]
Right, thanks for the speedy response! (I guess it makes sense to keep the issue open until it's fixed? Otherwise feel free to close)

**Comment by tobixen** [comment]
I thought the same, so I didn't close it :-)
